# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: BUG-EMAIL-ADDR <EMAIL@ADDRESS>\n"
"POT-Creation-Date: yyyy-mm-dd hh:mm+0000\n"
"PO-Revision-Date: 2016-10-04 12:58+0100\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
"X-Generator: Eazy Po 0.9.5.3\n"

#: WPKG-GP-Client.py:119
msgid ""
"Update error detected\n"
"during system start up."
msgstr ""

#: WPKG-GP-Client.py:121
msgid "WPKG Error"
msgstr ""

#: WPKG-GP-Client.py:122
msgid "System start error"
msgstr ""

#: WPKG-GP-Client.py:130
msgid ""
"System should be updated!\n"
"\n"
"System wasn't updated in over {} days."
msgstr ""

#: WPKG-GP-Client.py:132
msgid "Attention!"
msgstr ""

#: WPKG-GP-Client.py:139
msgid "Check for updates"
msgstr ""

#: WPKG-GP-Client.py:140
msgid "System update"
msgstr ""

#: WPKG-GP-Client.py:142
msgid "Help"
msgstr ""

#: WPKG-GP-Client.py:145
msgid "Cancel shutdown"
msgstr ""

#: WPKG-GP-Client.py:148 WPKG-GP-Client.py:519
msgid "Close"
msgstr ""

#: WPKG-GP-Client.py:184
msgid "Could not load updates:"
msgstr ""

#: WPKG-GP-Client.py:185
msgid "Update Error"
msgstr ""

#: WPKG-GP-Client.py:190
msgid "UPD:"
msgstr ""

#: WPKG-GP-Client.py:191
msgid "NEW:"
msgstr ""

#: WPKG-GP-Client.py:192
msgid "REM:"
msgstr ""

#: WPKG-GP-Client.py:193
msgid "DOW:"
msgstr ""

#: WPKG-GP-Client.py:199
msgid "Update(s) available:"
msgstr ""

#: WPKG-GP-Client.py:205
msgid "No Updates"
msgstr ""

#: WPKG-GP-Client.py:224
msgid ""
"Registry Error\n"
"\n"
"No access to necessary registry key."
msgstr ""

#: WPKG-GP-Client.py:230
msgid ""
"Reboot required!\n"
"\n"
"A reboot is required before the system\n"
"can be updated again.\n"
"Reboot now?"
msgstr ""

#: WPKG-GP-Client.py:237
msgid "System will reboot now!"
msgstr ""

#: WPKG-GP-Client.py:245
msgid "System Update"
msgstr ""

#: WPKG-GP-Client.py:260
msgid "WPKG-GP Client - Help"
msgstr ""

#: WPKG-GP-Client.py:299
msgid ""
"Close all open Applications, it is possible that programs will be closed "
"without a warning and system could reboot without further confirmation."
msgstr ""

#: WPKG-GP-Client.py:302
msgid "Attention"
msgstr ""

#: WPKG-GP-Client.py:309
msgid "Current Progress:"
msgstr ""

#: WPKG-GP-Client.py:312
msgid "Shutdown after update"
msgstr ""

#: WPKG-GP-Client.py:315
msgid "Open WPKG Log"
msgstr ""

#: WPKG-GP-Client.py:317
msgid "Update"
msgstr ""

#: WPKG-GP-Client.py:318 WPKG-GP-Client.py:371
msgid "Cancel"
msgstr ""

#: WPKG-GP-Client.py:345
msgid ""
"WPKG is currently running,\n"
"please wait a few seconds and try again."
msgstr ""

#: WPKG-GP-Client.py:351
msgid "2. Warning"
msgstr ""

#: WPKG-GP-Client.py:352
msgid ""
"Close all open programs!\n"
"\n"
"The System could restart without further confirmation!\n"
"\n"
"Continue?"
msgstr ""

#: WPKG-GP-Client.py:372
msgid ""
"System update in progress!\n"
"\n"
" Canceling this Progress could result in installation issues.\n"
"Cancel?"
msgstr ""

#: WPKG-GP-Client.py:399
msgid ""
"WPKG-GP Client detected a active VPN Connection using Cisco Anyconnect.\n"
"This could result in slow upgrade progress and updates for the AnyConnect\n"
"Software will be blocked.\n"
"Continue?"
msgstr ""

#: WPKG-GP-Client.py:456
msgid "WPKG-GP process aborted."
msgstr ""

#: WPKG-GP-Client.py:464
msgid "WPKG-GP Notification"
msgstr ""

#: WPKG-GP-Client.py:467
msgid ""
"WPKG-GP is currently running a task.\n"
"Retry later."
msgstr ""

#: WPKG-GP-Client.py:470
msgid "The update server could not be reached."
msgstr ""

#: WPKG-GP-Client.py:473
msgid ""
"The system was rejected from the server to execute an update!\n"
"Contact your IT department for further information."
msgstr ""

#: WPKG-GP-Client.py:476
msgid ""
"You are not authorized to execute a wpkg update!\n"
"Contact your IT department for further information."
msgstr ""

#: WPKG-GP-Client.py:479
msgid "Can't connect to the wpkg-gp service."
msgstr ""

#: WPKG-GP-Client.py:482
msgid "Unknown problem occured."
msgstr ""

#: WPKG-GP-Client.py:487
msgid "WPKG-GP process finished, restart necessary!"
msgstr ""

#: WPKG-GP-Client.py:489
msgid "WPKG-GP process finished."
msgstr ""

#: WPKG-GP-Client.py:491
msgid "Error detected during update"
msgstr ""

#: WPKG-GP-Client.py:496
msgid ""
"Reboot required!\n"
"\n"
"For the completion of the installation(s), a reboot is required.\n"
"Reboot now?"
msgstr ""

#: WPKG-GP-Client.py:502
msgid "System will reboot in %TIME% seconds."
msgstr ""

#: WPKG-GP-Client.py:510
msgid "System will shutdown in %TIME% seconds."
msgstr ""

#: WPKG-GP-Client.py:517
msgid "No System changes."
msgstr ""

#: WPKG-GP-Client.py:563
msgid "Can't open config file \"{}\"!"
msgstr ""

#: WPKG-GP-Client.py:570
msgid "An instance of WPKG-GP Client is already running!"
msgstr ""

#: WPKG-GP-Client.py:576
msgid "WPKG-GP Client requires at least version {} of the WPKG-GP Service."
msgstr ""